src/js/renderShabad.js

```diff
@@ -1,16 +1,37 @@
-function renderShabad(gurbani) {
+function renderShabad(gurbani,nav) {
   document.body.classList.remove("loading");
-  $shabad.appendChild(h('div', { class: 'shabad-container' }, [ baani(gurbani), ]));
+  let footnav = '';
+  if(typeof nav != "undefined") {
+    let link        = navLink(nav);
+    let pagination  = [];
+    if (typeof nav.previous != "undefined") {
+      pagination.push(h('a', { href: link + nav.previous }, '&laquo;'));
+    }
+
+    if (typeof nav.next != "undefined") {
+      pagination.push(h('a', { href: link + nav.next }, '&raquo;'));
+    }
+    footnav = h('div', { class: 'pagination'}, pagination);
+  }
+
+  $shabad.appendChild(h('div', { class: 'shabad-container' }, [ baani(gurbani), footnav]));
   $.each(prefs.displayOptions, function(index, option) {
     document.getElementById(option).click();
   });
   $.each(prefs.shabadToggles, function(index, option) {
     document.getElementById(option).click();
   });
   $controls.classList.remove('hidden');
+
+
 }
 
-function metaData(data) {
+function navLink(nav,source) {
+    if(nav.type == 'shabad') { return 'shabad?id='; }
+    else if(nav.type == 'ang') { return 'ang?source=' + source + '&ang='; }
+}
```

Comment:
Thanks for the PR

Could you follow the code style. Also, I would encourage you to use template strings & switch cases, as they make the code much more readable and concise. I guess we can rewrite this to also accept `ang`. What do you think ?

```diff
- function navLink(nav,source) {
+ function navLink (nav, source) {
-    if(nav.type == 'shabad') { return 'shabad?id='; }
-    else if(nav.type == 'ang') { return 'ang?source=' + source + '&ang='; }
+  switch (nav.type) {
+    case 'shabad': return 'shabad?id=';
+    case 'ang': return `ang?source=${source}&ang=`;
+  }
+}
```